### DISCUSSION TITLE
CMS

### DISCUSSION THREAD

**Discussion by pi43r** [lumeland/lume#452] [General]
Hey :)

I tried lume (and deno) for the first time in the last couple of days and it's really great!
Was just wondering if there is a way to connect a simple CMS for clients that are not as tech savvy?
I'm thinking of something like https://decapcms.org (formerly netlifycms)

For myself I'm planning to use obsidian as a markdown editor which is enough of a cms for me

**Comment by oscarotero** [comment]
Yes, there's the Netlify plugin (https://lume.land/plugins/netlify_cms/) to integrate the cms more easily.
